client/dashboard/app/context.tsx

```diff
@@ -58,19 +63,22 @@ export const APP_CONTEXT_DEFAULT_CONFIG: AppConfig = {
 	LoadingLogo: undefined,
 	supports: {
 		overview: false,
-		sites: false,
+		sites: false as const,
```

Comment:
I wonder why this change is needed 🤔

Reply:
It was needed before https://github.com/Automattic/wp-calypso/pull/106682